app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ProjectsController < ApplicationController
+  @@MAX_VOTES = 3
```

Comment:
if its a constant it should just be MAX_VOTES = 3.

you dont need the @@. @@ makes it a class variable which is not what a constant is really